services/checkin/service/auth_service.go

```diff
@@ -0,0 +1,55 @@
+package service
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/HackIllinois/api/common/apirequest"
+	"github.com/HackIllinois/api/services/rsvp/config"
+	"github.com/HackIllinois/api/services/rsvp/models"
+	"net/http"
+)
+
+/*
+	Add Attendee role to user with auth service
+*/
+func AddAttendeeRole(id string) error {
+	user_role_modification := models.UserRoleModification{ID: id, Role: "Attendee"}
+
+	body := bytes.Buffer{}
+	json.NewEncoder(&body).Encode(&user_role_modification)
+
+	status, err := apirequest.Put(config.AUTH_SERVICE+"/auth/roles/add/", &body, nil)
+
+	if err != nil {
+		return err
+	}
+
+	if status != http.StatusOK {
+		return errors.New("Auth service failed to update roles")
+	}
+
+	return nil
+}
+
+/*
+	Remove Attendee role from user with auth service
+*/
+func RemoveAttendeeRole(id string) error {
```

Comment:
Why do we need this function? It doesn't seem to be used anywhere in this service.

Reply:
Fixed.